src/main/java/com/rackspace/salus/monitor_management/services/MonitorManagement.java

```diff
@@ -1216,7 +1216,8 @@ private static boolean labelSelectorMethodChanged(Monitor original, MonitorCU ne
         .stream()
         // filter to keep resources that are not in the given exclusion set
         .filter(resourceDTO -> excludedResourceIds == null ||
-            !excludedResourceIds.contains(resourceDTO.getResourceId()))
+            !excludedResourceIds.stream().map(String::toLowerCase).collect(Collectors.toList()).contains(resourceDTO.getResourceId().toLowerCase()))
```

Comment:
Can you change this to `toSet` in order to retain the tiny optimization I had done with `excludedResourceIds`?
```suggestion
            !excludedResourceIds.stream().map(String::toLowerCase).collect(Collectors.toSet()).contains(resourceDTO.getResourceId().toLowerCase()))
```